Samples/Tutorial Samples/WinMLPIXSample/WinMLPIXSample/WinMLHelper.cpp

```diff
@@ -0,0 +1,146 @@
+#include "pch.h"
+#include "PixRedistMiddleware.h"
+#include "WinMLHelper.h"
+
+void LoadModel()
+{
+    // load the model
+    printf("Loading modelfile '%ws' on the '%s' device\n", modelPath.c_str(), deviceName.c_str());
+    DWORD ticks = GetTickCount();
+    model = LearningModel::LoadFromFilePath(modelPath);
+    ticks = GetTickCount() - ticks;
+    printf("model file loaded in %d ticks\n", ticks);

```

Comment:
is the timing data interesting for this sample?

Reply:
Maybe, if we can get timing data in out PIX capture as well?

Reply:
ok